ui/components/MesheryPatterns.js

```diff
@@ -750,6 +750,7 @@ function MesheryPatterns({
     setInfoModal({
       open: false,
     });
+    fetchPatterns(page, pageSize, search, sortOrder, visibilityFilter);
```

Comment:
@dragon-slayer875 thanks for raising PR, this change will always pull the latest pattern after closing info modal, which will make unnecessary API calls, instead we want to fetch latest changes after user clicks on Save button in info modal which will ensure that we fetch latest changes when change is made only

Reply:
@Yashsharma1911 I have updated the PR.
I made a new function which returns the latest fetchPatterns. The said function can be passed any future child components to update data on any further actions.

Reply:
Makes sense, although eventually as we will migrate to RTK we will not require to do this, because RTK always fetch latest changes on mutating data

@sudhanshutech FYI as you are tracking RTK migration

Reply:
@Yashsharma1911 I did think about RTK but wasn't sure how many changes I was allowed to introduce without prior discussions. But I am glad that we will soon be migrating to a more novel solution.